packages/sdk/src/auth.ts

```diff
@@ -162,10 +170,12 @@ export class AuthManager {
     { audience }: { audience?: string } = {
       audience: this.audience
     }
-  ): Promise<string> {
+  ): Promise<string | undefined> {
```

Comment:
their contract changed, not 100% that this wont have downstream consequences

Reply:
Specifically, if I'm reading the code right, this method
- Triggers a login
- Saves the result of that login to the cache
- Reads the access_token out of the cache

So if its undefined, it presumably means the token wasn't able to be read from the cache, probs indicating that the cache isnt readable